### ISSUE TITLE
Specify month for "start" and "end"

### ISSUE THREAD

**Issue by biologisturiel** [nstrayer/datadrivencv#24]
Hello, my CV is looking great! Also it seems wonderful to be able to update it from the drive.
My query is:
Is it possible to add the month to the year?
I have tried to put "2015-02" or "Feb2015". In none of the cases has it worked.
The error is as follows:

`Error: Problem with `mutate()` input `start`. 
x Result 2 must be a single string, not a character vector of length 0
ℹ Input `start` is `(function (.x, .f, ...) ...`.
Backtrace: ...`

**Comment by nstrayer** [comment]
This is something that I have been meaning to put in. The reason it doesn't work right now is the order of the entries is decided by the end date which is treated as an integer.

There are two ways that come to mind to get around this. 
- A new column called "current" that overrides any end date 
- Adding a sorting date column that just gives relative sorting of all entries and then whatever is put in start and end is copied as-is to the sections. 

My like of the first option is it doesn't require people to make large changes to their current data format... Maybe both options could be available via a function argument. 

@biologisturiel, do you have an opinion on one or the other being better?

**Comment by nstrayer** [comment]
This should be fixed in #25. Now you can provide your date in pretty much any format that you desire as long as it is month-year or plain year. Closing but will re-open if the update doesn't work. 

You can update to the newest version without wiping your rmd with

```
devtools::install_github("nstrayer/datadrivencv")
datadrivencv::use_datadriven_cv(
  full_name = ...
  which_files = c("cv_printing_functions.r")
)
```
